index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome-Books</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+
+<body>
+
+    <header>
+
+        <nav class="nav_menu">
+            <h1 class="firsttitle">Awesome books</h1>
+            <ul class="navlist">
+                <li><a class="navitem1" href="#list">List</a></li>
+                <li><a class="navitem2" href="#form">Add New</a></li>
+                <li><a class="navitem3" href="#footer">Contact</a></li>
+            </ul>
+        </nav>
+        <p id="dateTime"></p>
```

Comment:
- [ ] It would be better if we display the data and time in our web app. Currently we are not displaying the Luxon date-time functionality
![Screenshot from 2022-12-06 05-00-05](https://user-images.githubusercontent.com/46388113/205820140-20b1ec81-1e8a-4206-8ba2-3e9f9b026a3f.png)